packages/manager/modules/user-account/src/account.module.js

```diff
@@ -0,0 +1,44 @@
+import angular from 'angular';
+
+import '@ovh-ux/manager-core';
+import '@uirouter/angularjs';
+import 'angular-translate';
+import '@ovh-ux/ng-at-internet';
+import 'angular-ui-bootstrap';
+
+import '@ovh-ux/ui-kit';
+import '@ovh-ux/ui-kit/dist/css/oui.css';
+
+import './css/source.less';
+
+import ngOvhDedicatedUniverseComponents from '@ovh-ux/ng-ovh-dedicated-universe-components';
```

Comment:
```suggestion
import '@ovh-ux/ng-ovh-dedicated-universe-components';
```
as it's a peer dependency